data/json/items/gun/9mm.json

```diff
@@ -910,13 +910,13 @@
     "//": "Tilelist whitelist for pistols.",
     "type": "GUN",
     "name": { "str": "Glock duty pistol" },
-    "description": "Designed for all shooters, this offering from the Glock gun company is a full-sized 9x19mm pistol marketed towards law-enforcement and military users.",
+    "description": "Designed for all shooters, this offering from the Glock gun company is a full-sized 9x19mm pistol marketed towards law-enforcement and military users.  Because of long magazine well, it can't fit magazines shorter than the standard 17-round ones.",
```

Comment:
```suggestion
    "description": "Designed for all shooters, this offering from the Glock gun company is a full-sized 9x19mm pistol marketed towards law-enforcement and military users.  Because of the long magazine well, it can't fit magazines shorter than the standard 17-round ones.",
```